operator/src/main/java/org/keycloak/operator/v2alpha1/KeycloakController.java

```diff
@@ -97,8 +98,7 @@
 
         if (status.equals(kc.getStatus())) {
             return UpdateControl.noUpdate();
-        }
-        else {
+        } else {
```

Comment:
Since we are super early in development I thought it was a good idea to improve a little the consistency of the style (since it's not enforced ref: https://github.com/keycloak/keycloak/discussions/9227 ).

But, ok, I can leave this out.